samples/deepseek-r1-webgpu/src/llm.js

```diff
@@ -37,15 +37,15 @@ export class LLM {
   end_thinking_token_id = 0;
 
   async init() {
+    // Model config file and files used by the tokenizer are always downloaded locally.
+    env.allowRemoteModels = false;
```

Comment:
if you set ` env.allowRemoteModels` then all resources include `onnx models` will be downloaded locally, is that what we want to ?